### PR TITLE
docs(MAINTAINERS): add documentation describing project maintainers

### PR DESCRIPTION
**Pull request by carmstrong**

### CODE REVIEW

**Comment by mboersma** [comment]
LGTM.

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+project from a great one.
+
+# Deis maintainers
+
+Deis has two groups of maintainers in addition to our beloved Benevolent Dictator for Life.
+
+## BDFL
+
+Deis follows the timeless, highly efficient and totally unfair system known as [Benevolent dictator
+for life](http://en.wikipedia.org/wiki/Benevolent_Dictator_for_Life). Gabriel Monroy (@gabrtv), as
```
Comment (bacongobbler) [inline MAINTAINERS.md]: handles like @gabrtv aren't being properly handled in markdown. Maybe we should use direct links instead? Something like [Gabriel Monroy](https://github.com/gabrtv)

Comment (carmstrong) [inline MAINTAINERS.md]: Yeah, I did this elsewhere and forgot about him

Comment (bacongobbler) [inline MAINTAINERS.md]: This is also consistent with the Core Maintainers list below

**Comment by bacongobbler** [comment]
LGTM!

**Pull request merged**